plugin.video.manoto/default.py

```diff
@@ -0,0 +1,62 @@
+from builtins import next
+import xbmc, xbmcgui, os, xbmcaddon, xbmcplugin, urllib.request, urllib.error, urllib.parse, re
+
+def main():
+    __settings__ = xbmcaddon.Addon()
+    home = __settings__.getAddonInfo('path')
+    resources = os.path.join(home, 'resources')
+    addon_handle = int(sys.argv[1])
+
+    icons = {}
+    icons['Auto'] = xbmc.translatePath(os.path.join(resources, 'icon.png'))
+    icons['720'] = xbmc.translatePath(os.path.join(resources, 'hd.png'))
+    icons['1080'] = xbmc.translatePath(os.path.join(resources, 'fhd.png'))
+
+    streams = getStreamsFromPlayList()
+    bitrates = list(streams.keys())
+    bitrates.sort()
+    bitrates.reverse()
+
+    for bitrate in bitrates:
+         title = "Manoto TV ({quality})".format(quality=streams[bitrate][0])
+         li = xbmcgui.ListItem(title)
+         try:
+              li.setArt({'thumb': icons[streams[bitrate][2]]})
+         except IndexError:
+              li.setArt({'thumb': icon})
+         xbmcplugin.addDirectoryItem(handle=addon_handle, url=streams[bitrate][1], listitem=li, isFolder=False)
+
+    xbmcplugin.endOfDirectory(addon_handle)
+
+def getStreamsFromPlayList():
+    base_url = "https://d2rwmwucnr0d10.cloudfront.net/"
```

Comment:
this link is a bit suspect. Who is in charge of this, where does the link come from?

Reply:
it's from https://www.manototv.com/live, this plugin is known to them and it was suggested to use the hard-coded URL in a previous conversation with @keithah and @narimangharib.